kolibri/content/management/commands/exportcontent.py

```diff
@@ -0,0 +1,50 @@
+import logging as logger
+import os
+
+from django.db.models import Sum
+from kolibri.tasks.management.commands.base import AsyncCommand
+
+from ...content_db_router import using_content_database
+from ...models import File
+from ...utils import paths, transfer
+
+logging = logger.getLogger(__name__)
+
+
+class Command(AsyncCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument("channel_id", type=str)
+        parser.add_argument("destination", type=str)
+
+    def handle_async(self, *args, **options):
+        channel_id = options["channel_id"]
+        data_dir = os.path.realpath(options["destination"])
+        logging.info("Exporting content for channel id {} to {}".format(channel_id, data_dir))
+
+        with using_content_database(channel_id):
+            files = File.objects.all()
+            total_bytes_to_transfer = files.aggregate(Sum('file_size'))['file_size__sum']
+
+            with self.start_progress(total=total_bytes_to_transfer) as overall_progress_update:
+
+                for f in files:
+
+                    filename = f.get_filename()
+                    srcpath = paths.get_content_storage_file_path(filename)
+                    dest = paths.get_content_storage_file_path(filename, datafolder=data_dir)
+
+                    # if the file already exists, add its size to our overall progress, and skip
+                    # TODO(jamalex): could do md5 checks here instead, to be ultra-safe
```

Comment:
This one was less of a "MUST DO" and more of a "IF WE WANT TO DO MD5 CHECKING, IT WOULD GO HERE" note, for future reference. :)

Adding md5 checking here would be easy, but it's not necessarily a good idea, given the substantial performance penalty. I can just remove the TODO...